미션 1

butomop opened the pull request:
ArticleController
get "/" -> index.html
get "articles/new/" -> article-edit.html

butomop committed:
test

butomop committed:
[Start]

[Add]
ArticleController
- get "/" -> index
- get "articles/new" -> article-edit

templates
- article-edit.html
- index.html

butomop committed:
[Add]
HelloWorldController
- get,post "/helloworld" -> index

[Modify]
index.html -> using thymeleaf templates

butomop commented on src/main/resources/templates/index.html:
```diff
@@ -61,7 +61,7 @@
         <div class="welcome-text-container">
             <div class="welcome-txt">
-                <h1 class="text-white border bottom font-size-35 padding-5">Welcome Brown!</h1>
+                <h1 class="text-white border bottom font-size-35 padding-5" th:text="${blogName} ?:'Welcome Brown!'"/>
```
기존에 "/"로 접근 시 parameter가 없어서 블로그 이름이 뜨지않음
-> 기본값설정

butomop committed:
[Modify]
Get,PostMapping -> RequestMapping

boorownie commented on src/main/java/techcourse/myblog/HelloWorldController.java:
```diff
@@ -1,8 +1,15 @@
 @Controller
 public class HelloWorldController {
 
+    @RequestMapping(value = {"/helloworld","/"}, method = { RequestMethod.GET, RequestMethod.POST })
```
POST 메소드를 조회 시 사용할 필요는 없을 것 같은데...

boorownie commented on src/main/resources/templates/article-edit.html:
```diff
@@ -0,0 +1,135 @@
+                        </li>
+                        <li role="separator" class="divider"></li>
+                        <li class="text-dark">
+                            <a href="../static/index.html">
```
파일 이동 시 불필요하게 붙은거 같은데...

boorownie reviewed (changes requested):
테스트 코드를 구현 하나도 안했네요
ArticleControllerTests를 이용해서 테스트 해보세요 :)

butomop committed:
[Modify]
HelloWorldController
- remove wrong request parameter

article-edit
- fix invalid link

butomop committed:
[Add]
post "articles" -> article.html

butomop committed:
[Modify]
RequestParam -> Article

[Add]
Article getter

butomop committed:
[Add]
add article

butomop committed:
[Add]
add article id

butomop committed:
[Modify]
using template

thymeleaf cache - false

butomop committed:
[Add]
template files

butomop committed:
[Modify]
index list view -> descending order list

butomop committed:
[Add]
article edit
 - put "/article/{articleId}"

butomop committed:
[Add]
article delete -> delete "/article/{articleId}"
